[Bug]: release CI が失敗

### web-serial-rxjs バージョン

@gurezo/web-serial-rxjs@0.2.0

### RxJS バージョン

rxhs@7.8.x

### ランタイム

Browser (Web App)

### ブラウザ / バージョン

Chrome v146.x

### OS

macOS

### フレームワーク

Vanilla JS/TS

### Bundler / ビルドツール

_No response_

### 何が起きましたか？

git push origin v0.2.0 したら、エラーになった

### 再現手順

git push origin v0.2.0

### 期待される動作

CIをパスして、リリースされる

### ログ / エラースタック

```shell
Run npm -v
  npm -v
  npm i -g npm@^11.5.1
  npm -v
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    PNPM_HOME: /home/runner/setup-pnpm/node_modules/.bin
10.9.7
npm error code MODULE_NOT_FOUND
npm error Cannot find module 'promise-retry'
npm error Require stack:
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/rebuild.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/index.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/index.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/node_modules/libnpmfund/lib/index.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/lib/utils/reify-output.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/lib/utils/reify-finish.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/lib/commands/install.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/lib/npm.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/lib/cli/entry.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/lib/cli.js
npm error - /opt/hostedtoolcache/node/22.22.2/x64/lib/node_modules/npm/bin/npm-cli.js
npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2026-04-08T12_20_43_209Z-debug-0.log
```

### チェックリスト

- [x] Chromium 系ブラウザで試しました（Web Serial API 対応ブラウザ）
- [x] README の Quick Start を確認しました

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -23,13 +23,9 @@ jobs:
         with:
           node-version: "22"
 
-      # ★ここが肝：Trusted Publishing(OIDC)に必要な npm 11.5.1+ に上げる
-      - name: Upgrade npm for Trusted Publishing
+      - name: Show bundled npm version
         shell: bash
-        run: |
-          npm -v
-          npm i -g npm@^11.5.1
-          npm -v
+        run: npm -v
 
       - run: pnpm install --frozen-lockfile
       - run: pnpm test
@@ -85,7 +81,7 @@ jobs:
           npm config delete _authToken || true
 
           cd packages/web-serial-rxjs
-          npm publish --access public --provenance
+          pnpm dlx npm@11.5.1 publish --access public --provenance
 
       - name: Create GitHub Release
         uses: softprops/action-gh-release@v1

